analyzer_test.go

```diff
@@ -9,45 +9,55 @@ import (
 	"os"
 	"path/filepath"
 	"testing"
+
+	"github.com/Masterminds/semver"
+	"github.com/golang/dep/test"
 )
 
-func TestDeriveManifestAndLock(t *testing.T) {
-	dir, err := ioutil.TempDir("", "dep")
+func TestAnalyzerInfo(t *testing.T) {
```

Comment:
I am -1 on tests like this. You wrote the same code on both sides and are asserting they are equal.